### ISSUE TITLE
whatsapp reactions not received on second device

### ISSUE THREAD

**Issue by gilcu3** [d99kris/nchat#219]
**Description**:

While reactions are received correctly on a device, once logged-in in another device the reactions are not shown.

**How to reproduce it**:

Receive whatsapp messages in a computer then open a client in another computer.

**Environment**:
- Version: 4.71
- OS / distro: Archlinux

**Comment by d99kris** [comment]
Hi @gilcu3 - I have not seen this issue. Can you please share a bit more? Are you running nchat on both these computers? If not, what is the other app used?

One limitation with whatsapp is that nchat cannot fetch "old" reactions, so only reactions sent after you updated to an nchat version supporting reactions, will be shown in nchat. Not sure if this is the issue observed, but just wanted to mention it in case.

**Comment by gilcu3** [comment]
Hi, both were `nchat` updated with the last master branch. I just tried doing it again on a single computer, which should [produce the same effect] (closing app, sending reactions with phone, opening app to check) and I was not able to reproduce it. Therefore, I am closing this unless I am able to reproduce it again. Maybe it was as you said, the old version received those messages and when I updated the new version could not receive them again. Thank you for the great support!
